composer.lock

```diff
@@ -4,27 +4,26 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "e918eb157af24afe7c378bc37aa8ec7c",
```

Comment:
Zas composer update :( Takže se updatnula půlka vesmíru. `composer update co/chci --with-dependencies`

Reply:
nahrál jsem to přes `composer require keboola/db-extractor --update-with-dependencies` přes update to úplně nešlo kdy je to dev verze

Reply:
Aha OK, tak uvidíme, až to bude relasnuté...